chia/consensus/blockchain.py

```diff
@@ -126,13 +130,11 @@ def shut_down(self):
         self._shut_down = True
         self.pool.shutdown(wait=True)
 
-    async def _load_chain_from_store(self) -> None:
+    async def _load_chain_from_store(self, blockchain_dir):
```

Comment:
type annotation

Reply:
I think I was under the impression that we don't normally mark-up `None` return types. In fact, I think most functions that don't return anything we don't explicitly say `-> None`. I realize though, that mypy can't assume this means it doesn't return anything..